src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -172,21 +240,21 @@ public function get_grid_days( $date = null, $force = false ) {
 			} else {
 				$start = tribe_beginning_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
 				$end   = tribe_end_of_day( $day->format( Dates::DBDATETIMEFORMAT ) );
-				/*
-				 * We want events overlapping the current day, by more than 1 second.
-				 * This prevents events ending on the cutoff from showing up here.
-				 */
-				$day_query = tribe_events()
-					->set_found_rows(true)
-					->by_args( $repository_args )
-					->where( 'date_overlaps', $start, $end, null, 2 )
-					->per_page( $events_per_day )
-					->order_by( $order_by, $order );
-				$day_event_ids = $day_query->get_ids();
-				$found     = $day_query->found();
-
-				$this->grid_days_cache[ $day_string ]       = (array) $day_event_ids;
-				$this->grid_days_found_cache[ $day_string ] = (int) $found;
+
+				$results_in_day = array_filter( $day_results, static function( $event ) use ( $start, $end ) {
+					return (
+						( $start >= $event->start_date && $end <= $event->end_date )
+						|| ( $start >= $event->start_date && $start <= $event->end_date && $end >= $event->end_date )
+						|| ( $start < $event->start_date && $end >= $event->end_date )
+						|| ( $start < $event->start_date && $end <= $event->end_date && $end >= $event->start_date )
+					);
+				} );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 605552cff3117e3ab140d6c885594c14</sub>